core/modules/update/tests/update.test

```diff
@@ -281,7 +281,7 @@ class UpdateCoreTestCase extends UpdateTestHelper {
    */
   function testGetUpdaterFromDirectory() {
     module_load_include('inc', 'installer', 'installer.manager');
-    // Extract our stub sample backdop zip archive to temporary.
+    // Extract our stub sample backdrop zip archive to temporary.
```

Comment:
```suggestion
    // Extract our stub sample Backdrop zip archive to temporary.
```

I think the usage here is "Backdrop" as the proper noun (even though the file name is lower-case).